compiler/fir/cones/src/org/jetbrains/kotlin/fir/types/ConeAttributes.kt

```diff
@@ -65,7 +66,7 @@ class ConeAttributes private constructor(attributes: List<ConeAttribute<*>>) : A
     private inline fun perform(other: ConeAttributes, op: ConeAttribute<*>.(ConeAttribute<*>?) -> ConeAttribute<*>?): ConeAttributes {
         if (this.isEmpty() && other.isEmpty()) return this
         val attributes = mutableListOf<ConeAttribute<*>>()
-        for (index in indices) {
+        for (index in this.typeRegistry.indices.union(other.typeRegistry.indices)) {
```

Comment:
`indicies` here is a reference to property of companion object of `ConeAttributes` (which is a `typeRegistry` for any `ConeAttributes` object), so there is no need in this change

Reply:
Yeah, all of this happened due to my misunderstanding on union/intersect types. Reverted.